x/interchainstaking/keeper/delegation.go

```diff
@@ -280,14 +280,16 @@ func (k *Keeper) MakePerformanceDelegation(ctx sdk.Context, zone *types.Zone, va
 func (k *Keeper) FlushOutstandingDelegations(ctx sdk.Context, zone *types.Zone, delAddrBalance sdk.Coin) error {
 	var pendingAmount sdk.Coins
 	exclusionTime := ctx.BlockTime().AddDate(0, 0, -1)
+	var coinsToFlush sdk.Coins
 	k.IterateZoneReceipts(ctx, zone, func(_ int64, receiptInfo types.Receipt) (stop bool) {
 		if (receiptInfo.FirstSeen.After(exclusionTime) || receiptInfo.FirstSeen.Equal(exclusionTime)) && receiptInfo.Completed == nil {
 			pendingAmount = pendingAmount.Add(receiptInfo.Amount...)
 		}
 		return false
 	})
 
-	coinsToFlush, hasNeg := sdk.NewCoins(delAddrBalance).SafeSub(pendingAmount...)
```

Comment:
Why has this been reverted. It was this originally and we remove the var declaration and just went with := instead.

Reply:
i cpoied from the actual PR - nightmare https://github.com/ingenuity-build/quicksilver/pull/433/files#diff-f7f3e35ef9410a1ee6aa87107f7af45ae8f1c7239ea5b94df450f8a1fd359122R439

Reply:
i just saw removing

Reply:
didnt wanted to touch any logic so copy pasted